nncf/openvino/engine.py

```diff
@@ -62,10 +64,13 @@ class OVNativeEngine(Engine):
     to infer the model.
     """
 
-    def __init__(self, model: ov.Model):
+    def __init__(self, model: ov.Model, use_fp32_precision: bool = False):
```

Comment:
```suggestion
    def __init__(self, model: ov.Model, use_fp32_precision: bool = True):
```
Is it possible?

Reply:
Yes, it allows us to get the same results for SPR and VNNI. My question is: do we really need such a parameter, or is it better to use inference_precision: Type.f32 for all cases?

Reply:
Done